src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -107,6 +107,7 @@ public partial class SslStream : AuthenticatedStream
         public override int Read(byte[] buffer, int offset, int count) { return default(int); }
         public override long Seek(long offset, System.IO.SeekOrigin origin) { return default(long); }
         public override void SetLength(long value) { }
+        public virtual System.Threading.Tasks.Task ShutdownAsync() { return default(System.Threading.Tasks.Task); }
```

Comment:
Thanks @weshaggard : will follow the process as soon as I can get it to build / test using the new package.